test/tsconfig.json

```diff
@@ -0,0 +1,22 @@
+{
+    "compilerOptions": {
+      "rootDir": "..",
+      "outDir": "../dist/test",
+      "target": "es2017",
+      "types": ["node", "jest"],
+      "module": "commonjs",
+      "lib": [
+        "dom",
+        "es7"
+      ]
+    },
+    "include": [
+      "../src/**/*.ts",
+      "**/*.ts"
+    ],
+    "exclude": [
+      "../node_modules",
+      "**/*.unittest.ts"
```

Comment:
eh?.. Wouldn't this exclude all the tests? Is this tsconfig not used then?..

Reply:
I questioned this too but it seemed to work.  I took this config from layouts which also had it.  I removed it and didn't notice anything break so we will keep it that way